class InternalError(Exception):
    """Custom exception class."""
    pass

class FetchError(InternalError):
    """Custom exception class."""
    pass

